generators/server/types.d.mts

```diff
@@ -41,6 +42,7 @@ type DatabaseTypeSqlApplication = (
 ) & {
   devDatabaseType: string;
   prodDatabaseType: string;
+  devDatabaseTypeMysql: string;
```

Comment:
This is required? Should be an boolean and autocalculated

Reply:
@mshima Apologies for the mistake about the string type 😱 
However, as for the declaration, if it is not present, it results in a lint error.